### ISSUE TITLE
[docs] Really needs to mention that it does NOT work from the `working-directory` of the job

### ISSUE THREAD

**Issue by douglasg14b** [actions/upload-artifact#541]
### What files would you like to change?

The readme maybe? 

### What are your suggested changes?

This took actual ages to figure out, but turns out this action does not respect the `working-directory` default, but the readme doesn't make direct mention of it (Ctrl+F `working-directory`).

This should be made abundantly clear since it's a pretty big "gotcha"

**Comment by praneetloke** [comment]
@douglasg14b 

> but turns out this action does not respect the working-directory default

I am not sure if there is a job-level default for working directory in GH actions though? I am not seeing an option to do that in the docs. I do see a [step-level working-directory](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory) and the default that you can set for the [`run` step](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunworking-directory) but I don't think you are referring to either of those?

**Comment by robacarp** [comment]
@praneetloke 

As [documented](https://docs.github.com/en/actions/using-jobs/setting-default-values-for-jobs#setting-default-shell-and-working-directory), you can assign a default working directory for a job like this:

```yaml
jobs:
  ci_job:
    defaults:
      run:
        working-directory: ci_path
```

**Comment by praneetloke** [comment]
Yeah it's only for the `run` step. It's not for _all_ steps to use.

**Comment by robacarp** [comment]
Oh, sure enough. Wow that did not even occur to me, transparent as it is in the implementation.

**Comment by TheBeruriahIncident** [comment]
Currently the README says "Relative paths are rooted against the current working directory". I realize that strictly speaking, that is correct, but it's quite misleading as you may have configured a working directory shortly before, and the README does not refer to the working directory you expect.
